.github/workflows/live-test.yml

```diff
@@ -0,0 +1,33 @@
+on:
+  push:
+    branches:
+      - master
+
+name: live-test
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
```

Comment:
Would prefer that this is in the main test workflow since it compiles once or twice and then reuses the compiled test binary